citus-enterprise.spec

```diff
@@ -302,6 +305,7 @@ done < "$secret_files_list"
 %files
 %defattr(-,root,root,-)
 %doc CHANGELOG.md
+%doc vendor/safestringlib/LICENSE
```

Comment:
Should we use the renamed file here instead? `safestringlib-LICENSE`

Reply:
yes, thanks